packages/map/src/circle/index.stories.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { select, withKnobs } from '@storybook/addon-knobs';
+import config from '../../../../config';
```

Comment:
@peterhass did adding that alias work?

Reply:
using a alias would cause eslint to complain

```
error: Unable to resolve path to module 'config' (import/no-unresolved) at src/circle/index.stories.jsx:3:20:
  1 | import React from 'react';
  2 | import { select, withKnobs } from '@storybook/addon-knobs';
> 3 | import config from 'config';
```

Didn't want to add a exception to eslint config so I kept it like it is now